trough/client.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, rethinkdb_trough_db_url, promotion_interval=None):
         '''
         parsed = doublethink.parse_rethinkdb_url(rethinkdb_trough_db_url)
         self.rr = doublethink.Rethinker(
-                servers=parsed.hosts, db=parsed.database)
+                servers = parsed.hosts, db = parsed.database)
```

Comment:
From [PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements):
```

Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:

# Correct:
def complex(real, imag=0.0):
    return magic(r=real, i=imag)
# Wrong:
def complex(real, imag = 0.0):
    return magic(r = real, i = imag)
```